src/map/script.c

```diff
@@ -24559,6 +24559,26 @@ static BUILDIN(getcalendartime)
 	return true;
 }
 
+static BUILDIN(setfavoriteitem)
+{
+	struct map_session_data *sd = script_rid2sd(st);
+	int idx = script_getnum(st, 2);
+	int value = script_getnum(st, 3);
+
+	if (sd == NULL) {
+		ShowError("buildin_setfavoriteitem: no player attached.\n");
+		return false;
+	} else if (idx < 0 || sd->status.inventorySize <= 0) {
```

Comment:
check "idx" with max size?
because this may cause crash if idx is not proper.
e.g:
```
idx = 100
sd->status.inventorySize = 60
condition: if (100 < 0 || 60 < 0) => False
then it would go below at line 24574 and crash.

Reply:
done